operator-pipeline-images/operatorcert/static_tests/community/bundle.py

```diff
@@ -47,6 +97,7 @@ def run_operator_sdk_bundle_validate(
         bundle.root,
         "--select-optional",
         test_suite_selector,
+        f"--optional-values=k8s-version={kube_version_for_deprecation_test}",
```

Comment:
How does the new parameter communicate the deprecated API? Is this part of the structured json output or is it just a warning log?

Reply:
it is part of a json output